### PR TITLE
feat: [OCA-1418] - Adds September 2024 Marketplace apps

### PR DESCRIPTION
**Pull request by josephcardillo**
## Description 📝
Adds new Marketplace apps for September 2024
Apache Spark Cluster

## Changes  🔄
List any change relevant to the reviewer.
- updates oneClickAppsv2.ts
- adds SVG public assets

## Target release date 🗓️
September 16, 2024

## Preview 📷

apache_spark_screenshot1
apache_spark_screenshot2

### CODE REVIEW

**Review by mjac0bs — APPROVED** [review]
Thanks @josephcardillo! Confirmed that Apache Spark is showing up with the correct icons, a cluster chip, no typos, link addresses that redirect correctly (or will on 9/16). Approving pending minor updates to the changeset.

Screenshot 2024-09-03 at 9 59 05 AM

File: packages/manager/.changeset/pr-10874-added-1725380346411.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Added
+---
+
+Adds new Marketplace apps for September 2024 ([#10874](https://github.com/linode/manager/pull/10874))
```
[the remark this answers is not in the document]
Comment (dwiley-akamai) [reply packages/manager/.changeset/pr-10874-added-1725380346411.md:5]: I echo the above 👍🏾

**Review by dwiley-akamai — APPROVED** [review]
`Apache Spark Cluster` appears in the "New Apps" section when pointing at prod and the app drawer looks good ✅

File: packages/manager/.changeset/pr-10874-added-1725380346411.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (josephcardillo) [reply packages/manager/.changeset/pr-10874-added-1725380346411.md]: Thank you for pointing that out. I just updated it. I'll be sure to follow those best practices going forward. My apologies for missing that!

**Pull request merged**